Test/git-issues/git-issue-604a.dfy

```diff
@@ -0,0 +1,15 @@
+// RUN: %dafny "%s" /compile:3 /compileTarget:cs > "%t"
+// RUN: %dafny "%s" /compile:3 /compileTarget:js > "%t"
+// RUN: %dafny "%s" /compile:3 /compileTarget:java > "%t"
+// RUN: %dafny "%s" /compile:3 /compileTarget:go > "%t"
+// RUN: %diff "%s.expect" "%t"
+
+method Inv() returns (x: int) {
```

Comment:
Incorporate this test in the file git-issue-604.dfy, rather than using a separate file for it.